### PR TITLE
Added floating point data types support in SEEP

### PR DESCRIPTION
**Pull request by elkasoapy**

### CODE REVIEW

File: seep-common/src/main/java/uk/ac/imperial/lsds/seep/api/data/Type.java [hunk 1]
```diff
@@ -360,4 +360,42 @@ public Object randomValue() {
...
+		}
+
+		@Override
+		public boolean isVariableSize() {
+			return false;
+		}
+
+		@Override
+		public Object defaultValue() {
+			return 0.0f;
```
Comment (pgaref) [inline seep-common/src/main/java/uk/ac/imperial/lsds/seep/api/data/Type.java]: It is a Double value so for the default you need something like:  **0.0d**

**Comment by elkasoapy** [comment]
Fixed now, thanks Panagiotis!

**Comment by raulcf** [comment]
How is this different from Matthias commit?

https://github.com/raulcf/SEEPng/commit/035057591f8bf71fc58b996748004f716ca02056

Is the difference that this one also supports double?
If so, has it been tested with double data? It'd be great to add a unit test, there are already a bunch that can be used as example.

**Comment by pgaref** [comment]
The main difference is that Matthias commit introduced only the Type but ITupple should be also extended to retrieve the float/double from the api - also need support for those types in the Schema. @elkasoapy is now extending the SchemaTest to demonstate the functionality :)

**Comment by raulcf** [comment]
Ah! great, thanks!
As soon as the test is there safe to merge

**Comment by elkasoapy** [comment]
Done! Please let me know if you find any other issues with this.

**Comment by pgaref** [comment]
Works for me :+1:

**Comment by raulcf** [comment]
great thanks!

**Pull request merged**